doc/Language/objects.pod6

```diff
@@ -19,30 +19,28 @@ object. That includes literals, types (type objects), code and containers.
 
 =head1 Using Objects
 
-You can use objects by calling methods on them. To call a method on an
-expression, add a dot, followed by the method name:
+To call a method on an object, add a dot, followed by the method name:
 
 =for code :allow<B L>
 L<say> "abc"B<.L<uc>>;
 # ABC
 
 This calls the L<C<uc>|uc> method on C<"abc">, which is an object of type
-L<C<Str>|Str>. If you need to supply arguments to the method, add
-parentheses after the method name with the arguments inside the parentheses:
+L<C<Str>|Str>. To supply arguments to a method, add parentheses after the
+method, with arguments inside parentheses:
 
 =for code :allow<B L>
 my $formatted-text = "Fourscore and seven years ago...".L<indent>B<(8)>;
 
 C<$formatted-text> now contains the above text, but indented 8 spaces.
 
-Multiple arguments simply require a comma between them:
+Separate multiple arguments with commas:
```

Comment:
I saw this as an opportunity to do away with the passive voice. I'll make your change.